src/data/sources.json

```diff
@@ -33,19 +33,19 @@
     "name": "BBC news",
     "site": "https://www.bbc.com/news/science-environment-56837908",
     "base": "https://www.bbc.com",
-    "region":""
+    "region":"Americas"
```

Comment:
This one's European. The first 'B' stands for British.

Brexit didn't get us thrown out of Europe all together, just the trade union! :joy: :joy: